pkg/plugin/util/testdata/exampleFile.txt

```diff
@@ -1 +1 @@
-exampleTarget
\ No newline at end of file
+exampleTargetexampleCodeexampleCode
```

Comment:
I think we need to make sure this change doesn't get committed right? I *think* there is a test that fails if this gets committed as is